shard.yml

```diff
@@ -1,5 +1,5 @@
 name: passkit
-version: 0.1.0
+version: 0.2.0
 crystal: 0.35.1
```

Comment:
Should extend the supported crystal range here.

Reply:
I will make the bump a separate PR

Reply:
May as well  `">= 0.35.1"` and save the effort.